### DISCUSSION TITLE
scRNA label transfer with SCTransform normalized expression values

### DISCUSSION THREAD

**Discussion by joseverdezoto** [GreenleafLab/ArchR#1870] [Questions / Documentation]
Hello,

Thanks for developing this awesome package. I had a question regarding transfer of scRNA labels to scATAC-seq data. We created a scRNA atlas integrating multiple datasets where we annotated immune and non-immune cell subtypes. We normalized each library using SCTransform (as it is known to perform better compared to the previous log normalization). However, I'm wondering if there'd be an issue with the addGeneIntegrationMatrix() since I believe this function extracts data from the RNA assay within a Seurat object. In our case, normalized data would be stored within an "SCT" assay instead. Would there be a way to get around this?

We'd really appreciate your input. 

Jose

**Comment by rcorces** [comment]
No - this is not possible. Last I checked, label transfer with SCT was not recommended by the creators of Seurat either. If that understanding is out of date, feel free to follow up.
